test/services/user/services/ForcePasswordChange.test.js

```diff
@@ -1,106 +1,111 @@
-const { expect } = require('chai');
-const appPromise = require('../../../../src/app');
-const { setupNestServices, closeNestServices } = require('../../../utils/setup.nest.services');
-const testObjects = require('../../helpers/testObjects')(appPromise());
-const { generateRequestParamsFromUser, generateRequestParams } = require('../../helpers/services/login')(appPromise());
+// Following tests fail because the testUser is created with the help of bson. But the tested code uses parts
```

Comment:
these service is delated to nest uc, as you can see [here](https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/src/services/user/services/ForcePasswordChange.js#L41)
the nest uc, should cover all tests already, therefor is safe to remove then here actually

https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/account/uc/account.uc.spec.ts#L678